src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.MergeAssetManifests/PublishVerticalManifest.cs

```diff
@@ -10,9 +10,9 @@
 using System.Linq;
 using System.Xml.Linq;
 
-namespace Microsoft.DotNet.SourceBuild.Tasks.PublishVerticalManifest
+namespace Microsoft.DotNet.SourceBuild.Tasks
```

Comment:
This isn't a source-build task. Consider renaming this to something like `Microsoft.DotNet.UnifiedBuild.Tasks`.

Reply:
I see that this would force you to rename everything, so I'm ok with keeping this name for now.